pkg/threatest/matchers/elasticsearch/elasticsearch.go

```diff
@@ -0,0 +1,137 @@
+package elasticsearch
+
+import (
+	"fmt"
+	"strings"
+	log "github.com/sirupsen/logrus"
+	"encoding/json"
+	"errors"
+)
+
+func FilterByUuidPresence(alerts []ElasticsearchQueryHit, uuid string) []ElasticsearchQueryHit {
+	var filteredAlerts []ElasticsearchQueryHit
+	var containsUuid bool
+	for _,alert := range alerts {
+		containsUuid = false
+		for _,v := range alert.Source {
+			if strings.Contains(v.(string), uuid) {
+				containsUuid = true
+				break
+			}
+		}
+		if containsUuid {
+			filteredAlerts = append(filteredAlerts, alert)
+		}
+	}
```

Comment:
```suggestion
	for _, alert := range alerts {
		containsUuid := false
		for _, v := range alert.Source {
			if strings.Contains(v.(string), uuid) {
				containsUuid = true
				break
			}
		}
		if containsUuid {
			filteredAlerts = append(filteredAlerts, alert)
		}
	}
```
Spacing in the for loops and moving the variable inside the loop